client/app/widgets/Notifications/__tests__/Notifications.spec.jsx

```diff
@@ -12,7 +12,7 @@ const data = {
   fetch_notifications: ['Notification 1'],
 };
 
-const component = (
+const getComponent = () => (
```

Comment:
I'm confused about why you made this into a function and renamed this. This new function isn't even being used in this file. I think we can keep what was originally here.

Reply:
Sure :+1: